### DISCUSSION TITLE
Feature Request: Limit which TickTick projects are sync'd to Obsidian

### DISCUSSION THREAD

**Discussion by CodeCity** [thesamim/TickTickSync#124] [Ideas]
I use TickTick for both work and personal. I not only use it as a task manager, but I also to help me remember things. If I get an idea or if I want to remember something for later, I put it in a specific list. 

For example, along with various projects, I have a list for TV shows and movies to WATCH. If I read an article or see a trailer about a particular show or movie, I put it in the list. I have another list for GROCERTIES, so when I run out of something I can put it in the GROCERIES list and I won't forget to replenish that item the next time I go to the grocery store. 

For work, there are similar lists that do not need to be sync'd. I have a List that is a template for a project that gets repeated throughout the year. There are tasks there, but it mostly sits idle until that type of project is needed, it then gets copied and renamed so I don't have to recreate the tasks every time. 

I don't need to sync these lists/projects with Obsidian. It would be helpful if all the projects were displayed under the plugin Options, and then we could select which lists/projects we would like to sync. We could then go in and select new projects or lists that we need to sync while others could sit idle.

**Comment by thesamim** [comment]
This has been implemented.
